packages/terra-clinical-item-view/docs/README.md

```diff
@@ -0,0 +1,52 @@
+# Terra Clinical Item View
+
+The Terra ClinicalItemView component allows displays to be organized into rows and column and themed, while providing means to add accessory elements.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-clinical-item-view`
+  - `yarn add terra-clinical-item-view`
+
+## React Usage
+
+The React component can be initialized like so:
+```
+import ClinicalItemView from 'terra-clinical-item-view';
+
+<ClinicalItemView
+  layout="oneColumn"
+  textEmphasis="default"
+  isTruncated=true
+  accessoryAlignment="top"
+  leftAccesory=<img alt="Graphic" />
+  displays={[<ClinicalItemView.Display text="display 1" />, <ClinicalItemView.Display text="display 2" />]}
+  comment=<ClinicalItemView.Comment text="comment" />
+/>
+
+<Display
+  text="display text"
+  textStyle="attention"
+  isTruncated=true
```

Comment:
nit, If a prop is boolean, you can just write it as `isTruncated` instead of `isTruncated=true`